src/Pages/License/Generate/CustomerDetails.jsx

```diff
@@ -1,12 +1,60 @@
+import React from "react";
```

Comment:
Separate groups for react, material ui and internal components. Put line breaks between each group

Reply:
done for all